### PR TITLE
Add notes and warnings regarding AWS Signature Versions

### PR DESCRIPTION
**Pull request by TheOnlyRew**
Also, clean up some command lists, typos, and bad links.

### CODE REVIEW

**Comment by TheOnlyRew** [comment]
This commit may need to be duplicated in the riakcs/2.0.0 branch.

File: source/languages/en/riakcs/cookbooks/configuration/Configuring-an-S3-Client.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ This tutorial will show you how to use [s3cmd](http://s3tools.org/s3cmd)
 as an S3 client. While it won't cover all of the client's features, it
 will show you how to create a configuration and run some basic commands.
 
+>**Warning: s3cmd Signature Version**
+> If you are using s3cmd version 1.5.0 or greater you will need to append the
```
Comment (laurenrother) [inline source/languages/en/riakcs/cookbooks/configuration/Configuring-an-S3-Client.md]: In order for there to be a space in between you need an extra > between the Warning heading and the body text.

File: source/languages/en/riakcs/tutorials/fast-track/Testing-the-Installation.md [hunk 2]
```diff
@@ -22,14 +22,24 @@ We can install it on Ubuntu by typing:
 sudo apt-get -y install s3cmd
 ```
 
-For OS X users, either use the package manager of your preference or
-download the S3 cmd package at [[http://s3tools.org/download]]. You will
-need to extract the `.tar` file, change directories into the folder, and
-build the package. The process should look something like this:
+>**s3cmd versions**
```
Comment (laurenrother) [inline source/languages/en/riakcs/tutorials/fast-track/Testing-the-Installation.md]: Same here.

File: source/languages/en/riakcs/tutorials/fast-track/Testing-the-Installation.md [hunk 3]
```diff
@@ -62,6 +72,12 @@ installation steps.
 
 ## Interacting with Riak CS via S3cmd
 
+>**Warning: s3cmd Signature Version**
+> If you are using s3cmd version 1.5.0 or greater you will need to append the
```
Comment (laurenrother) [inline source/languages/en/riakcs/tutorials/fast-track/Testing-the-Installation.md]: And here. Sorry.

**Comment by laurenrother** [comment]
:+1:  :tada:

**Pull request merged**